### ISSUE TITLE
feat(MatChipEditInput): Add aria-label attribute to MatChipEditInput directive

### ISSUE THREAD

**Issue by clamli** [angular/components#31198]
### Feature Description

Add a general aria-label to MatChipEditInput like "editable chip", and consider removing the `tabindex="-1"` set by this directive.

### Use Case

This will resolve the `aria-input-field-name` violation when the chip turns into editable mode.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
